src/components/dialogs/filters-editor.js

```diff
@@ -144,127 +119,102 @@ const FiltersEditor = ({ filters, onChange }) => {
                         }
-                    />
-                </Grid>
-            </Grid>
-
-            <Grid container direction="row" spacing={1}>
-                <Grid item xs={12} sm={3}>
-                    <h3>
-                        {' '}
-                        <FormattedMessage id="equipmentType" />
-                    </h3>
-                </Grid>
-                <Grid item xs={12} sm={9}>
-                    <FormControl>
-                        <CustomNativeSelect
-                            id="demo-customized-select-native"
-                            value={filters.equipmentType}
-                            onChange={handleEquipmentType}
-                            input={<BootstrapInput />}
-                        >
-                            {Object.values(EquipmentTypes).map((val) => (
-                                <option value={val} key={val}>
-                                    {intl.formatMessage({ id: val })}
-                                </option>
-                            ))}
-                        </CustomNativeSelect>
-                    </FormControl>
-                </Grid>
-            </Grid>
-
-            <Grid container direction="row" spacing={1}>
-                <Grid item xs={12} sm={3}>
-                    <h3>
-                        <FormattedMessage id="Countries" />
-                    </h3>
-                </Grid>
-                <Grid item xs={12} sm={9}>
-                    <CustomAutocomplete
-                        id="select_countries"
-                        value={filters.countries}
-                        multiple={true}
-                        onChange={(event, newValue) => {
-                            handleCountrySelection(newValue);
+                        InputProps={{
+                            style: {
+                                borderRadius: '0 4px 4px 0',
+                            },
                         }}
-                        input={<BootstrapInput />}
-                        options={Object.keys(countriesList.object())}
-                        style={BootstrapInput.input}
-                        getOptionLabel={(code) => countriesList.get(code)}
-                        renderInput={(props) => <TextField {...props} />}
-                        renderTags={(val, getTagsProps) =>
-                            val.map((code, index) => (
-                                <Chip
-                                    id={'chip_' + code}
-                                    size={'small'}
-                                    label={countriesList.get(code)}
-                                    {...getTagsProps({ index })}
-                                />
-                            ))
-                        }
                     />
                 </Grid>
-            </Grid>
-        </div>
+            </FormControl>
+            <FormControl fullWidth margin="dense">
```

Comment:
You should remove this FormControl component here.
The `equipmentType` is duplicated otherwise

Reply:
We need to duplicate `equipmentType` here, first one will be the displayed value, second one is here to erase the border  of the field behind the `equipmentType`

Reply:
ok then